upcloud-api.postman_collection.json

```diff
@@ -523,6 +523,41 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "List object storages",
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Authorization",
+						"value": "{{authorization}}"
+					},
+					{
+						"key": "Accept",
+						"value": "application/json"
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					}
+				],
+				"url": {
+					"raw": "https://api.upcloud.com/1.3/object-storage/",
+					"protocol": "https",
+					"host": [
+						"api",
+						"upcloud",
+						"com"
+					],
+					"path": [
+						"1.3",
+						"object-storage",
+						""
+					]
+				}
+			},
+			"response": []
```

Comment:
Would want the rest of the endpoints here too. So that all the CRUD methods can be tested through postman as well.

Reply:
@PopoSensei see conversation up above https://github.com/UpCloudLtd/upcloud-go-api/pull/71#discussion_r521933313

Reply:
Oh. If we decide to not support, then forget this :)